lib/compilers/sway.ts

```diff
@@ -0,0 +1,245 @@
+import path from 'path';
```

Comment:
Can you add an appropriate Copyright header here please (copy/paste from another and make sure it's updated to 2025) thanks

Reply:
sure thing!